renderer/components/editor/options/export-select.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {DropdownArrowIcon} from '../../../vectors';
+
+const OptionType = PropTypes.shape({
+  label: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired
+});
+
+const Select = ({options}) => (
+  <select>
+    {options.map(option => <option key={option.value}>{option.label}</option>)}
+    <style jsx>{`
+    select {
+      position: absolute;
+      height: 100%;
+      width: 100%;
+      opacity: 0;
+    }
+  `}</style>
+  </select>
+);
+
+Select.propTypes = {
+  options: PropTypes.arrayOf(OptionType).isRequired
+};
+
+const ExportSelect = ({options, label}) => {
+  const hasOptions = options && options.length !== 0;
```

Comment:
`options &&` is moot here are it's marked as required in the proptypes.

`options.length !== 0` => `options.length > 0`